ui/src/app/(home)/_components/after-login.tsx

```diff
@@ -107,8 +107,18 @@ const AfterLogin = async () => {
                                         width={pageInfoItem.icon.width}
                                         height={pageInfoItem.icon.height}
                                         className={`mr-5 mb-4 max-w-${pageInfoItem.icon.width} h-auto`}/>
-                                    {pageInfoItem.number
-                                        && <span className="text-[2.5rem] text-text-color">{pageInfoItem.number}</span>}
```

Comment:
`{ pageInfoItem.number !== null` might work too, not really sure how your 0 comes up as unstyled.